internal/xds/translator/testdata/out/xds-ir/accesslog-multi-cel.listeners.yaml

```diff
@@ -0,0 +1,220 @@
+- accessLog:
+  - filter:
+      andFilter:
+        filters:
+        - extensionFilter:
+            name: envoy.access_loggers.extension_filters.cel
+            typedConfig:
+              '@type': type.googleapis.com/envoy.extensions.access_loggers.filters.cel.v3.ExpressionFilter
+              expression: response.code >= 400
+        - extensionFilter:
+            name: envoy.access_loggers.extension_filters.cel
+            typedConfig:
+              '@type': type.googleapis.com/envoy.extensions.access_loggers.filters.cel.v3.ExpressionFilter
+              expression: request.url_path.contains('v1beta3')
+        - responseFlagFilter:
+            flags:
+            - NR
+    name: envoy.access_loggers.file
+    typedConfig:
+      '@type': type.googleapis.com/envoy.extensions.access_loggers.file.v3.FileAccessLog
+      logFormat:
+        textFormatSource:
+          inlineString: |
+            [%START_TIME%] "%REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)% %PROTOCOL%" %RESPONSE_CODE% %RESPONSE_FLAGS% %BYTES_RECEIVED% %BYTES_SENT% %DURATION% %RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)% "%REQ(X-FORWARDED-FOR)%" "%REQ(USER-AGENT)%" "%REQ(X-REQUEST-ID)%" "%REQ(:AUTHORITY)%" "%UPSTREAM_HOST%"
+      path: /dev/stdout
+  - filter:
+      andFilter:
+        filters:
+        - extensionFilter:
+            name: envoy.access_loggers.extension_filters.cel
+            typedConfig:
+              '@type': type.googleapis.com/envoy.extensions.access_loggers.filters.cel.v3.ExpressionFilter
+              expression: response.code >= 400
```

Comment:
Can "response.code" or "request.url_path" be used at the listener-level logger?

Reply:
never try that in istio, I'm not sure whether envoy know protocol at listener.